### ISSUE TITLE
Extension Broken on bolt 3.3^

### ISSUE THREAD

**Issue by jackiboy** [SahAssar/bolt-editcontentcustomtabs#6]
When using the extension on bolt V3.3 ^ I get the following Uncaught Exception.

```
ReflectionException in CustomEdit.php line 47: 
Method setCanUpload does not exist
```

*More details:*
```
         * We need to access these later down, but they are all private, so we
         * get a reflection of the parent.
         */
        $reflector = (new \ReflectionObject($this))->getParentClass();

        $setCanUpload = $reflector->getMethod('setCanUpload');
        $setCanUpload->setAccessible(true);

        $getPublishingDate = $reflector->getMethod('getPublishingDate');
        $getPublishingDate->setAccessible(true);

```
I am using bolt *3.3.1*

- Browser: Google Chrome
- OS: OSX El Capitan
- PHP: 7.1.1
- Server: Apache

**Comment by cofunin** [comment]
+1

**Comment by ezcoops** [comment]
+1

**Comment by ezcoops** [comment]
@SahAssar any chance you might be looking into this in the near future?

**Comment by SahAssar** [comment]
Sorry, I've been focused on my new(ish) job for the last year or so, so I haven't had that much time to do bolt development. I can't say when I will have time to work on it again, but if anyone wants to take a stab at fixing this I'd be happy to merge working PR's.

**Comment by JarJak** [comment]
+1

**Comment by JarJak** [comment]
@SahAssar could you at least update composer.json with `        "bolt/bolt": "^3.0, <3.3"`

**Comment by SahAssar** [comment]
@JarJak You know there is a way on github to propose changes, right?

**Comment by SahAssar** [comment]
Fixed by #7, thanks JarJak
